worker/src/handlers/post.ts

```diff
@@ -18,7 +18,10 @@ export async function handlePost(request: Request): Promise<Response> {
   }
 
   const storageKey = `${KV_PREFIX_UUID}:${payload.uuid}`;
-  const country = request.headers.get("cf-ipcountry");
+  let country = request.cf.country;
+  if (country === "US" && request.cf.regionCode) {
+    country = `${country}_${request.cf.regionCode}`;
```

Comment:
Shouldn't we add a new field for the region?

Reply:
It basically replaces the country for US, for other countries we don't use it

Reply:
If we add a region field, it will be easier to add regions for other countries in the future.